beacon-chain/sync/initial-sync/round_robin.go

```diff
@@ -124,7 +124,9 @@ func (s *Service) processFetchedDataRegSync(
 	blockReceiver := s.chain.ReceiveBlock
 	for _, blk := range data.blocks {
 		if err := s.processBlock(ctx, genesis, blk, blockReceiver); err != nil {
-			log.WithError(err).Debug("Block is not processed")
+			log.WithError(err).WithFields(logrus.Fields{
+				"blk": blk,
```

Comment:
`blk` as an object is too verbose imo. (e.g. 128 attestations)

Reply:
You are absolutely right. Should we log just `blk.Block.Slot`?